EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/attributeInput/StringAttributeInput.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2012-2017, Business Process Technology (BPT),
+ * http://bpt.hpi.uni-potsdam.de.
+ * All Rights Reserved.
+ *
+ *******************************************************************************/
+package de.hpi.unicorn.application.pages.input.generator.attributeInput;
+
+import de.hpi.unicorn.event.attribute.TypeTreeNode;
+import org.apache.log4j.Logger;
+
+public class StringAttributeInput extends AttributeInput {
+
+	private String value;
+
+	static final Logger logger = Logger.getLogger(StringAttributeInput.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') "logger" is the name of a field in "AttributeInput". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS2387)